src/plone/restapi/services/site/configure.zcml

```diff
@@ -0,0 +1,20 @@
+<configure
+    xmlns="http://namespaces.zope.org/zope"
+    xmlns:plone="http://namespaces.plone.org/plone"
+    >
+
+  <plone:service
+      method="GET"
+      factory=".get.SiteGet"
+      for="Products.CMFPlone.interfaces.IPloneSiteRoot"
+      permission="zope2.View"
+      name="@site"
+      />
+
+  <adapter
+      factory=".get.Site"
+      name="site"
+      />
```

Comment:
I don't understand why the `@site` endpoint is available as an expander. The information is the same for any item in the site and not contextual, so using it as an expander would just repeat sending data multiple times for no reason, right?

Reply:
@erral Yes, but why? The frontend should only need to load the site endpoint once, rather than once each time a content item is fetched, right?

Reply:
anyway, we can remove the expandable thing quite easily

Reply:
@erral I would say: remove it. It's also easy to re-add later if we remember a reason it's needed. (It's also not currently documented, so removing it helps move forward toward merging everything else.)

Reply:
Removed and pushed.